package.xml

```diff
@@ -12,8 +12,8 @@
     </lead>
     <date>2019-01-21</date>
     <version>
-        <release>0.10.0</release>
-        <api>0.10.0</api>
+        <release>0.11.0</release>
+        <api>0.11.0</api>
```

Comment:
We just need to update `<notes>` with the CHANGELOG stuff as well. :)

Reply:
Ohhhh 🤦‍♂️